hand-eye-calibration/full_hand_eye_sample.py

```diff
@@ -0,0 +1,486 @@
+"""
+Script to set up communication, generate dataset and perform hand-eye calibration.
```

Comment:
Should have more instructions for use, and perhaps list dependencies.
You forgot to update the README file, btw.

Reply:
Done